backend/server.js

```diff
@@ -538,6 +529,253 @@ app.post('/subscribeEmail', wrap(async (req, res) => {
 }));
 
 
+// finds the user with the login key that's been requested
+// if the user doesn't exist, return
+async function findMatchingUser(requestLoginKey) {
+  // Loop over the db
+  const users = await database.get('users');
+  if (!users) {
+    return null;
+  }
+
+  // Loop over all the users
+  for (const user of users) {
+    for (const loginKey of user.loginKeys) {
+      if (requestLoginKey === loginKey) {
+        return user;
+      }
+    }
+  }
+
+  return null;
+}
+
+
+// sends data to the database in the backend
+async function sendUserHelper(req, res) {
```

Comment:
Want rename this to something like `verifyRequestAndGetDbUser` ? 

I know this is totally contradicting stuff taught at neu but I feel like naming functions "helper" isn't very informative to what they actually do.